docs/manual/figures/value-subtyping.svg

```diff
@@ -1,791 +1,183 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This comment doesn't appear to have been addressed. The `@` signs are still hard to read because the boxes aren't large enough - can you make the boxes on `@ArrayLen` and `@ArrayLenRange` a little bigger so that the text fits? Thanks.